README.md

```diff
@@ -149,6 +149,30 @@ This has to be done only once.
 make style
 ```
 
+### Running the one-line-scan workflow locally
+
+#### Build the `ktf-one-line-scan` container
+
+* first, download the `one-line-scan` base image
+```bash
+docker build -t one-line-scan https://raw.githubusercontent.com/awslabs/one-line-scan/master/tools/Dockerfile
```

Comment:
This line is long, but arguably hard to split.

Reply:
Fixed.